### ISSUE TITLE
[Bug] Tasks disappears on scrolling

### ISSUE THREAD

**Issue by roman-yukhno-axon** [ANovokmet/svelte-gantt#234]
I noticed a problem: when there are more tasks on one line than can be displayed, they disappear when scrolled. As an example, I created 20 tasks for one line. At the same time, there are absolutely no errors in the browser console. What could it be? And how to solve this?

```
    const from = moment().subtract(10, 'day');
    const to = moment().add(1, 'month');

    const tasks = [];
    for (let i = 0; i < 15; i++) {
      const id = i.toString().padStart(5, '0');
      tasks.push({
        id: id,
        resourceId: '123456',
        from: from.clone().add(i, 'day'),
        to: to.clone().add(i, 'day'),
        label: `Task #${id}`
      });
    }

    this.gantt = new SvelteGantt({
      target: document.getElementById('gantt-element'),
      props: {
        ...this.options,
        layout: 'expand',
        tasks,
        rows: [{ id: '123456', label: 'Test test', height: null }],
      }
    });
```
![IntelliFinder](https://github.com/user-attachments/assets/7c662a17-a9c8-4ecb-8ecc-0b9cb50cc487)

**Comment by ANovokmet** [comment]
Looks like a bug with virtual scrolling + layouts, where visibleRows is calculated incorrectly when layout expands the row height. Ill try to fix it as soon as i can. I have no idea about any workarounds.

**Comment by roman-yukhno-axon** [comment]
I noticed that this happens when the first task is hidden behind the header

**Comment by ANovokmet** [comment]
This is weird, it happens only when there is just 1 row. So for a "workaround", you can add another row. It's going to be fixed in v4.4.1
